datasets/ONS-UK-trade-in-services/info.json

```diff
@@ -18,32 +19,36 @@
         "main_issue": 17,
         "columns": {
             "Period": {
-                "parent": "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod",
-                "value": "http://reference.data.gov.uk/id/{+period}",
-                "codelist": false
+                "type": "dimension",
+                "uri": "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod",
+                "value": "http://reference.data.gov.uk/id/{+period}"
             },
             "Country": {
-                "parent": "http://purl.org/linked-data/sdmx/2009/dimension#refArea"
+                "type": "dimension"
```

Comment:
You could make use of local codelists.
"new": {
     "codelist": "./codelists/country.csv-metadata.json"
    }